app/src/main/java/com/x8bit/bitwarden/ui/platform/components/indicator/BitwardenCircularProgressIndicator.kt

```diff
@@ -0,0 +1,15 @@
+package com.x8bit.bitwarden.ui.platform.components.indicator
+
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+/**
+ * A Bitwarden-styled [CircularProgressIndicator].
+ */
+@Composable
+fun BitwardenCircularProgressIndicator(
+    modifier: Modifier = Modifier,
+) {
+    CircularProgressIndicator(modifier = modifier)
+}
```

Comment:
What are we gaining here? Is this so we can theme this later if necessary?

Reply:
Yea, we need to apply our theme and we'll need to add a track.

The `CircularProgressIndicator` supports all of this but its nicer to have all of that in a single spot.

Reply:
Yeah that makes sense then.